scripts/python/packages/parthenon_tools/parthenon_tools/movie2d.py

```diff
@@ -97,8 +142,18 @@ def plot_dump(
 
 if __name__ == "__main__":
     addPath()
-    field = sys.argv[1]
-    files = sys.argv[2:]
+    args = parser.parse_args()
+    field = args.field
+    files = args.files
+    components = [0, 0]
+    if (args.tc is not None) and (args.vc is not None):
+        raise ValueError(
+            "Only one of --tensor-components and --vector-component should be set."
+        )
+    if args.tc is not None:
+        components = tc
```

Comment:
Does this need to be `components = args.tc`? Maybe I'm missing something.

Reply:
no you're right thanks for the catch